container/docker/factory.go

```diff
@@ -40,6 +41,7 @@ var ArgDockerEndpoint = flag.String("docker", "unix:///var/run/docker.sock", "do
 var DockerNamespace = "docker"
 
 // Basepath to all container specific information that libcontainer stores.
+// TODO: Deprecate this flag
```

Comment:
This is not yet deprecated. I was thinking of getting rid of this flag altogether. Attempting to be backwards compatible with that will only make the code messier.